src/plugins/console/public/application/hooks/use_send_current_request/send_request.test.ts

```diff
@@ -49,34 +49,43 @@ describe('sendRequest', () => {
     expect(mockedSendRequest).toHaveBeenCalledTimes(1);
   });
 
-  it('should send multiple requests', async () => {
-    mockedSendRequest.mockResolvedValue([
-      {
-        response: {
-          statusCode: 200,
+  describe('with multiple requests', function () {
```

Comment:
nit: please next time let's use one pattern for declare functions at least in one file. I don't really like it when we have different styles in the code